### DISCUSSION TITLE
Get the row height don't work ?

### DISCUSSION THREAD

**Discussion by MatthieuDelisle** [exceljs/exceljs#1873] [Q&A]
Hello, I want to get the height of a row, but it doesn't seem to work. It think it's not implemented yet.
It work for a column, but for a row it only work after setting a row's height
This image explain it all.

![image](https://user-images.githubusercontent.com/44803799/138274900-b142f02a-1ffd-410d-bd83-540b82775008.png)

Note that I want the current row height, not the "defaultRowHeight" of a worksheet.

Thank you for your help.

**Comment by Siemienik** [comment]
It's implemented and works great. The value `undefined` is the default value. In Excel, it is 15. It can be different through various operating systems (because it is calculated from the default font). Additionally, It can be changed globally for the specific workbook.

Lectures:
1. https://support.microsoft.com/en-us/office/change-the-column-width-and-row-height-72f5e3cc-994d-43e8-ae58-9774a0905f46
2. https://excelribbon.tips.net/T005663_Changing_Default_Row_Height.html
3. https://bettersolutions.com/excel/rows-columns/row-heights.htm

**Reply by MatthieuDelisle** [reply]
As I said,  I want the current row height, not the "defaultRowHeight" of a worksheet...

**Comment by yogithesymbian** [comment]
how about we have set `wrapText : true` then we need to `get height of the row` ? just because **we cant use wrap text on merge cell**.
my case is i have merge cell A1 - C1 , then wrapText doesnt work on merge cell
